spec/rubocop/formatter/formatter_set_spec.rb

```diff
@@ -83,6 +83,16 @@ module Formatter
           formatter_set.add_formatter('simple')
         end
 
+        around do |example|
+          begin
+            stdout = $stdout
+            $stdout = StringIO.new
```

Comment:
I misunderstood `$stdout` is reassigned, but it is replaced by `IO#reopen`.
https://github.com/tmm1/test-queue/blob/4a715e6c04c4e169481fa897431ad1ae52272470/lib/test_queue/runner.rb#L387

So we can use `STDOUT` instead of `$stdout`. Thanks!